dev/tasks/r/github.macos-linux.local.yml

```diff
@@ -46,6 +46,9 @@ jobs:
           # brew install sccache
           # remove cmake so that we can test our cmake downloading abilities
           brew uninstall cmake
+          # install GNU libtool to ensure that we simulate the CRAN macOS builders
+          brew install libtool
+          echo "/usr/local/opt/libtool/libexec/gnubin" >> $GITHUB_PATH
```

Comment:
The CRAN runners actually have `/Library/Frameworks/R.framework/Resources/bin/libtool` on their path it would appear, though that results in the same thing as we get here: a GNU libtool being found. 

Maybe this should actually "just" be changed to `echo "/Library/Frameworks/R.framework/Resources/bin" >> $GITHUB_PATH` instead?